src/api-service/__app__/onefuzzlib/workers/scalesets.py

```diff
@@ -875,6 +876,52 @@ def set_state(self, state: ScalesetState) -> None:
                 )
             )
 
+    def sync_auto_scale_settings(self) -> Optional[Error]:
+        logging.info("Trying to sync auto scale settings for scaleset %s" % self.scaleset_id)
+        
+        auto_scale_profile = get_auto_scale_profile(self.scaleset_id)
+        if auto_scale_profile is None:
+            auto_scale_profile_failed = Error(
+                code=ErrorCode.UNABLE_TO_FIND,
+                errors=["Failed to get auto scale profile for scaleset %s" % self.scaleset_id],
+            )
+            logging.error(auto_scale_profile_failed)
+            return auto_scale_profile_failed
+
+        minimum = auto_scale_profile.capacity.minimum
+        maximum = auto_scale_profile.capacity.maximum
+        default = auto_scale_profile.capacity.default
+        
+        scale_out_amount = 0
+        scale_out_cooldown = 0
+        scale_in_amount = 0
+        scale_in_cooldown = 0
+        
+        for rule in auto_scale_profile.rules:
+            scale_action = rule.scale_action
+            if scale_action.direction == "Increase":
+                scale_out_amount = scale_action.value
+                scale_out_cooldown = scale_action.cooldown
+            else if scale_action.direction == "Decrease":
+                scale_in_amount = scale_action.value
+                scale_in_cooldown = scale_action.cooldown
+            else:
+                pass
```

Comment:
We have a pretty 'standardized' auto scaling profile across all of our scalesets so the assumptions you made here are true. But I think if something changes (intentionally or not) in our auto scale profiles and we don't convert as expected, we could benefit from having a warning here that we would hopefully pick up in telemetry.